app/views/submissions/steps/population/_step03_before.html.haml

```diff
@@ -62,6 +62,10 @@
     columns</i>. Also, your organisation should ensure the unique naming of accession - so you never
     duplicate the <b>Plant accession</b> value in the context of the same <b>Originating organisation</b>.
 
+  %p
+    When including plant lines already defined in BIP it is best to skip all optional information. Otherwise, any
+    conflicts with existing data will prevent plant lines from being included in submitted population.
```

Comment:
However, the parser prevents no-species rows. So I had to supply correct Species value for an existing PL, to have it accepted. Either change the description here or, preferred, make rows with existing PLs acceptable without Species value.

Reply:
I made Species optional too.